App/Source/DownloadHandler.cpp

```diff
@@ -100,7 +100,7 @@ void DownloadHandler::InstallZip(std::string ZipPath, std::string Name, std::str
 	{
 		PrevMod.Remove();
 	}
-
+	if(ZipPath != "") { 
```

Comment:
imo, this would be better if it was an early return instead of a huge if statement.
```suggestion
	if (ZipPath == "")
	{
	    return;
	}
```